frameit/lib/frameit/device_types.rb

```diff
@@ -1,11 +1,13 @@
 module Frameit
   module Color
-    BLACK = "SpaceGray"
-    SILVER = "Slvr"
+    BLACK = "Space Gray"
```

Comment:
Currently frameit doesn't support iPhone 7 anyway, right? It will just use the iPhone 6 assets